base-template/about-base-template.md

```diff
@@ -1,8 +1,21 @@
 # About Base Template
 
+{Template author tip:
+
+* This about-base-template captures the structure and common elements that all downstream about-xxx-templates should follow.
+* It provides guidance for both template authors as well as document authors.
+* “Template author and template reviewer tips” should be removed from the final about-{doctype}-template.md. Other tips should remain.
+* Tweak other tips to align with the doctype you are writing for.
+
+}
+
+This document provides guidance related to the authoring of documents based on the {doctype}-template. It focuses on the “how”, “why” and where applicable, references background theory to justify the approaches recommended.
+
 ## Introduction
 
-This document is used to create a uniform, regulated template for producing multiple similarly structured documents, that is, as a base template to create any other template. The base template guides you through the steps to create a functional template that can be reused, adhered to, and followed consistently throughout.
+_{TBD: Introduction text below needs tweaking to align with revised base template structure.}_
+
+This document helps create a uniform, regulated template for producing multiple similarly structured documents, that is, as a base template to create any other template. The base template guides you through the steps to create a functional template that can be reused, adhered to, and followed consistently throughout.
```

Comment:
Hmmm, uniform and regulated and similarly structured seem to be synonyms for template. I wonder if we can get away with less?
```suggestion
This document is a base template that you can use to create any other template. The base template guides you through the steps to create a functional template that can be reused, adhered to, and followed consistently throughout.
```